lib/pharos/terraform/apply_command.rb

```diff
@@ -32,7 +32,10 @@ def tf_apply
 
       def pharos_up
         run_cmd "terraform output -json > .#{workspace}.json"
-        cmd = ['--tf-json', ".#{workspace}.json", '-c', config_yaml.filename]
+
+        cmd = @config_options || []
```

Comment:
I don't get this... why not an empty array always?

Reply:
`config_yaml.filename` is absolute and is there even when the default `cluster.yml` was used. The `@config_options` has the original param only if one was supplied.

Reply:
Of course it could be `cmd = []; cmd.concat(@config_options) if @config_options`